imports/ui/components/surveyForm/Question.js

```diff
@@ -157,17 +198,18 @@ export default class Question extends Item {
             disabled={this.isRefused() || disabled}
             checked={!this.isRefused() && v.value == value} // eslint-disable-line
             onChange={this.handleChange}
-          /> {v.label}
+          />{' '}
```

Comment:
feel free to use non breaking space `&nbsp;` in such cases

Reply:
I have used '\xa0' as '&nbsp;' was not working